server.js

```diff
@@ -363,32 +363,26 @@ const groupMessageHandler = async (message) => {
     );
   }
 
-  const result = await submitSearch(imageURL, responding_msg, message);
-  if (result.isAdult) {
-    await sendMessage(
-      message.chat.id,
-      "I've found an adult result 😳\nPlease forward it to me via Private Chat 😏",
-      {
-        reply_to_message_id: responding_msg.message_id,
-      }
-    );
+  await sendChatAction(message.chat.id, "typing");
 
-    return;
-  }
-  await sendMessage(message.chat.id, result.text, {
-    reply_to_message_id: responding_msg.message_id,
-    parse_mode: "Markdown",
-  });
+  const result = await submitSearch(imageURL, responding_msg, message);
 
   if (result.video && !messageIsSkipPreview(message)) {
     const videoLink = messageIsMute(message) ? `${result.video}&mute` : result.video;
     const video = await fetch(videoLink, { method: "HEAD" });
     if (video.ok && video.headers.get("content-length") > 0) {
```

Comment:
Sometimes the media server may refuse to generate video. Should also fallback to text message when video.ok is false or video content length is 0.